app/src/main/java/org/wikipedia/suggestededits/SuggestedEditsTasksFragment.kt

```diff
@@ -91,7 +96,12 @@ class SuggestedEditsTasksFragment : Fragment() {
 
     private fun onUserStatClicked(view: View) {
         when (view) {
-            contributionsStatsView -> showContributionsStatsViewTooltip()
+            contributionsStatsView -> {
+                showContributionsStatsViewTooltip()
+                if (!contributionObjects.isEmpty()) {
+                    startActivity(SuggestedEditsContributionsActivity.newIntent(requireActivity(), contributionObjects, userContributionsContinuation!!))
```

Comment:
For the sake of simplicity, I'm not sure we need to save & pass the user contributions from here to the new activity.  It's ok if the new activity makes the same API call again, but that way it will be much cleaner and more modular.